### DISCUSSION TITLE
Can't figure out how to get placeholder-url

### DISCUSSION THREAD

**Discussion by mikethecalamity** [renovatebot/renovate#30179] [Request Help] [answered]
### What would you like help with?

I would like help with my configuration

### How are you running Renovate?

Self-hosted

### If you're self-hosting Renovate, tell us which platform (GitHub, GitLab, etc) and which version of Renovate.

Gitlab

### Please tell us more about your question or problem

My Helm charts use a placeholder alias for the repository of some charts. I could not find any documentation on how to handle this, other than https://github.com/renovatebot/renovate/issues/4681, from 5 years ago, which makes it seem like it is possible. In my config, I tried putting the placeholder in `registryAliases` both at the root and under `helmv3` (see config example below). But neither seem to work. The logs still have `placeholder-url` (also below). Is there something I'm missing?

**Chart.yaml**
```yaml
dependencies:
  - name: my-chart
    version: 9.1.1
    repository: "@my-chart-registry"
```

**Config**
```js
registryAliases: {
  '@my-chart-registry': 'htttps://my.registry.com/charts'
}
helmv3: {
  registryAliases: {
    '@my-chart-registry': 'htttps://my.registry.com/charts'
  }
}
```

### Logs (if relevant)

<details><summary>Logs</summary>

```json
{
    "name": "renovate",
    "hostname": "runner-uwnp5jql-project-77108-concurrent-0",
    "pid": 20,
    "level": 20,
    "logContext": "ggOtyH0cxitg7iANIHOrq",
    "repository": "repo",
    "packageFiles": {
        "helmv3": [
            {
                "deps": [
                    {
                        "depName": "my-chart",
                        "currentValue": "9.1.1",
                        "skipReason": "placeholder-url",
                        "updates": [],
                        "packageName": "my-chart"
                    }
                ],
                "datasource": "helm",
                "packageFileVersion": "0.1.0",
                "lockFiles": [
                    "charts/my-project/Chart.lock"
                ],
                "packageFile": "charts/my-project/Chart.yaml"
            }
        ]
    },
    "msg": "Checking packageFiles for deprecated packages",
    "time": "2024-07-15T05:16:41.523Z",
    "v": 0
}
```

</details>

**Comment by RahulGautamSingh** [comment, marked as answer]
Drop the `@`

```
registryAliases: {
'@my-chart-registry': 'htttps://my.registry.com/charts' // nope
  'my-chart-registry': 'htttps://my.registry.com/charts' // yes
}
```
